scripts/spack/packages/spheral/package.py

```diff
@@ -29,6 +29,7 @@ class Spheral(CachedCMakePackage, CudaPackage, PythonPackage):
     variant('openmp', default=True, description='Enable OpenMP Support.')
     variant('docs', default=False, description='Enable building Docs.')
     variant('shared', default=True, description='Build C++ libs as shared.')
+    variant('caliper', default=False, description='Build Caliper support.')
```

Comment:
I don't think we need to make caliper/timers a variant in our spack package just yet. I think we can continue to always build it as a dependency. Locking in our host-configs to enabling / disabling timers seems explicit and I'd rather leave it as a CMake configuration option only so developer don't need to separate tpl-manager instances when building with and without timer support.

Reply:
So we want the our tpls to always have Caliper, but we want the ENABLE_TIMER option to control not only enabling the Caliper code inside of our timer macro layer but also if Spheral looks for Caliper as a tpl? If I am understanding your comment correctly.

Reply:
Yep, that is correct, if we aren't using timers we don't need to link or include caliper